src/components/imageBanner/index.js

```diff
@@ -2,8 +2,8 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
 
-import BannerTitle from '../bannerTitle'
-import BannerSubtitle from '../bannerSubtitle'
+import BannerTitle from '../BannerTitle'
+import BannerSubtitle from '../BannerSubtitle'
```

Comment:
Should these have changed? Probably the only reason this works is because macOS has a case-insensitive file system.

Reply:
my linter shouted at me because the files themselves are named in camelCase

Reply:
ignore me

Reply:
![image](https://user-images.githubusercontent.com/17087167/40511950-329fbad6-5f99-11e8-9a97-e73583767256.png)

Reply:
I renamed all the `PascalCase` folders and final names to `camelCase` to match the rest of the application and community convention.

This was done using `git mv` so the changes should be reflected by rebasing or merging master into your feature branch.